packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -340,7 +340,7 @@ export const messages = {
   CompactLayout: 'Compact Layouts',
   ConnectedApp: 'Connected Apps',
   ContentAsset: 'Content Assets',
-  CorsWhitelistOrigin: 'Cors Whitelist Origins',
+  CorsAllowlistOrigin: 'Cors Allowlist Origins',
   CspTrustedSite: 'Csp Trusted Sites',
```

Comment:
@LisandroFernandezSF Since this is the name of an entity, and therefore part of the public api I think this change will need to happen at a later date.